### PR TITLE
Wizard Animation

### PR DESCRIPTION
**Pull request by olaplassen**
It is now possible to create a wizard animation from the WizardTextrue Class. 
image

When a animation needs to be added, one can set the animation when needed like this: goodWizard.setAnimation(0f,0f, assets, WizardTextures.GoodWizardAttack1), where the last parameter is the desired animation.

### CODE REVIEW

File: core/src/main/kotlin/no/ntnu/beardblaster/screen/MenuScreen.kt [hunk 1]
```diff
@@ -88,10 +85,10 @@ class MenuScreen(
             game.setScreen<TutorialScreen>()
         }
         logoutBtn.onClick {
-            if (UserAuth().isLoggedIn()) {
+            /*if (UserAuth().isLoggedIn()) {
```
Comment (cherrybeans) [inline core/src/main/kotlin/no/ntnu/beardblaster/screen/MenuScreen.kt]: Should this be commented out when merging to main?

File: core/src/main/kotlin/no/ntnu/beardblaster/screen/MenuScreen.kt [hunk 2]
```diff
@@ -116,6 +113,6 @@ class MenuScreen(
 
     override fun dispose() {
         super.dispose()
-        UserData.instance.deleteObserver(this)
+        /*UserData.instance.deleteObserver(this)*/
```
Comment (cherrybeans) [inline core/src/main/kotlin/no/ntnu/beardblaster/screen/MenuScreen.kt]: Should this be commented out when merging to main?

**Review by cherrybeans — APPROVED** [review]
Look at uncommenting the stuff I commented on, then all good

File: core/src/main/kotlin/no/ntnu/beardblaster/screen/MenuScreen.kt (hunk 2 above)
Comment (olaplassen) [reply core/src/main/kotlin/no/ntnu/beardblaster/screen/MenuScreen.kt]: Yep, pushed a fix

**Pull request merged**